setup.py

```diff
@@ -99,7 +99,8 @@ def checkProgramIsInstalled(self, program, args, where_to_download,
                        "pyBigWig >=0.3.7",
                        "hicexplorer >= 2.1.1",
```

Comment:
There is no dependency to this HiCExplorer version. Please change it to hicmatrix and check the other dependencies as well.